src/Views/Filter/Subject.cshtml

```diff
@@ -1,10 +1,14 @@
 @model SubjectFilterViewModel
+@using GovUk.Education.SearchAndCompare.UI.Shared.Features
+@using Microsoft.AspNetCore.Mvc.Razor.Extensions
+@inject IFeatureFlags FeatureFlags
 
 @{
   ViewBag.Title = $"Find courses by subject";
   var selected = Model.FilterModel.SelectedSubjects;
   var isInWizard = ViewBag.IsInWizard == true;
   var errors = ViewBag.Errors as ErrorViewModel;
+  var showSenFilter = FeatureFlags.SearchSenCourses;
```

Comment:
I'm really nervous about implicitly typed variables that are used in razor ifs - nullable bools have bitten us twice! I appreciate that this is in fact a bool but can we either turn this line into `bool showSenFilter = ...` or line 95 into `@if (showSenFilter == true)`?

Reply:
Are you requesting a change @the-daniel-rothig or just thinking aloud? 🤓

Reply:
Gently suggesting a change 😊